terminal/src/com/jediterm/terminal/ui/TerminalPanel.java

```diff
@@ -970,6 +970,7 @@ public void initKeyHandler() {
     // cursor state
     private boolean myCursorIsShown; // blinking state
     protected Point myCursorCoordinates = new Point();
+    private CursorShape shape = CursorShape.BLINK_BLOCK;
```

Comment:
Could you please follow the code style and add my prefix to the field name?

Reply:
It is to better show that this is an object field, and not a method variable, or a parameter, or... So you don't have to use `this.` for disambiguation and avoid confusion with local variable for instance.